keycloak/README.md

```diff
@@ -0,0 +1,6 @@
+## Purpose
+
+This common action covers the one of the limitations of Terraform, such that there is no way to control of existing resources, and ensures the target Keycloak instances to be provisioned as:
```

Comment:
Verbiage is strange here, suggested "This common action covers one of the limitations of Terraform, such that there is no way to control existing resources, and it ensures the target Keycloak instances are provisioned as:"
(is this what you meant?)

Reply:
@thegentlemanphysicist I updated it as you suggested. thanks!